### PR TITLE
repeated words

### PR DESCRIPTION
**Pull request by paronicholas**
- [x] Top-level README “Table of Contents” is updated
- [x] Feature tasks for this challenge are completed
- [x] Unit tests written and passing
    - [x] “Happy Path” - Expected outcome
    - [x] Expected failure
    - [x] Edge Case (if applicable/obvious)
- [x] README for this challenge is complete
    - [x] Summary, Description, Approach & Efficiency, Solution, Resources
    - [x] Link to code
    - [x] Link to tests

### CODE REVIEW

File: code401challenges/src/main/java/RepeatedWord/RepeatedWord.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+public class RepeatedWord {
+  public static String repeatedWord(String sentence) {
+    HashTable<String, String> repeatTable = new HashTable<>(1024);
+    String[] sentArr = sentence.split(" ");
+    for(String word : sentArr) {
+      word = word.replaceAll("[^a-zA-Z]", "").toLowerCase();
+      if (repeatTable.has(word)) {
+        return word;
+      } else {
+        repeatTable.set(word, word);
```
Comment (evanslaton) [inline code401challenges/src/main/java/RepeatedWord/RepeatedWord.java]: Not that this actually matters, but booleans are extremely space efficient. If you have a hashmap where the value doesn't matter, using a boolean is a good idea: `repeatTable.set(word, true)`
